packages/backend/src/accounting/psql/ledger-transfer/index.test.ts

```diff
@@ -67,7 +65,8 @@ describe('Ledger Transfer Service', (): void => {
         )
 
         await expect(
-          ledgerTransferService.getAccountTransfers(
+          getAccountTransfers(
+            serviceDeps,
```

Comment:
It doesn't really matter, since in `container.start()` the objection models will be instantiated with `Model.knex(knex)`, so they don't need to pass in `knex` directly. 

I updated it to be explicit, however